### DISCUSSION TITLE
TypeError: cannot read property 'message' of null

### DISCUSSION THREAD

**Discussion by SamueleD78** [vfsfitvnm/frida-il2cpp-bridge#172] [Q&A]
As usual, thank you for your help, it's greatly appreciated ^^

Trying to call a metod of a class, i get this weird error. I would like to understand what it means.

The method prototipe (with renamed members) is:

```

static System.Void MethodName(System.IntPtr handle, SuccessCallbackWrapper onSuccess, FailureCallbackWrapper onFailure, System.String Keys, System.String Fields, DebugOption options);

```

In the same class i have other static methods without the "handle" param, and i'm able to call them flawlessly.

**Comment by vfsfitvnm** [comment]
I don't know, I think the problem occurs here: https://github.com/vfsfitvnm/frida-il2cpp-bridge/blob/3dce8b6c0f2d171403b213b759e2672b476efef8/src/il2cpp/structs/method.ts#L215

`e` (the error Frida throws) is null which is unexpected
